app/views/users/_user.html.slim

```diff
@@ -1,5 +1,5 @@
 tr
-  td= link_to user.email, user
+  td= link_to user.full_name, profile_path(user.profile)
```

Comment:
Here it was requested to use `natural_name` => `Firstname Lastname`.
`lastname, firstname` should only be used for clients and volunteers on index and show. 

applies to all changes of this pull request.